### ISSUE TITLE
Vizy field not visible in Form Template

### ISSUE THREAD

**Issue by bavocorteville** [verbb/formie#1478]
### Describe the bug

When navigating to the Form Template within Craft CMS, the Vizy field is not visible. As a result, users cannot interact with the field or enter any content.

<details><summary>Error message in the console</summary>

```
runtime-core.esm-bundler.js:244 TypeError: Cannot read properties of undefined (reading 'buttons')
    at Proxy.created (VizyInput.vue:277:38)
    at No (runtime-core.esm-bundler.js:173:36)
    at Yr (runtime-core.esm-bundler.js:182:21)
    at Mx (runtime-core.esm-bundler.js:3608:5)
    at MD (runtime-core.esm-bundler.js:3510:9)
    at ZA (runtime-core.esm-bundler.js:7387:9)
    at XD (runtime-core.esm-bundler.js:7298:9)
    at YA (runtime-core.esm-bundler.js:7220:11)
    at Q (runtime-core.esm-bundler.js:5542:13)
    at q (runtime-core.esm-bundler.js:5517:17)
oD @ runtime-core.esm-bundler.js:244
runtime-core.esm-bundler.js:244 TypeError: Cannot convert undefined or null to object
    at Function.entries (<anonymous>)
    at zte (GlobalAttributes.js:10:12)
    at Proxy.getExtensions (VizyInput.vue:327:44)
    at Proxy.mounted (VizyInput.vue:168:30)
    at runtime-core.esm-bundler.js:2757:40
    at No (runtime-core.esm-bundler.js:173:22)
    at Yr (runtime-core.esm-bundler.js:182:21)
    at Gm.t.__weh.t.__weh (runtime-core.esm-bundler.js:2731:29)
    at xp (runtime-core.esm-bundler.js:359:46)
    at Ae (runtime-core.esm-bundler.js:6281:9)
``` 
</details> 

I suspect that the error might be related to the initialization of TipTap within Formie and then separately for the Vizy field?

### Steps to reproduce

1. Go to the Formie plugin settings.
2. Navigate to the Form Templates section.
3. Select a Form Template.
4. Add a Vizy field to the Form Fields.
5. Save the changes.
6. Navigate to a form that uses this Form Template.

### Form settings

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

### Craft CMS version

4.4.14

### Plugin version

2.0.31

### Multi-site?

No

### Additional context

Vizy version: 2.1.6

**Comment by engram-design** [comment]
This is a tricky one, as the two Vue apps don't seem to like playing together nicely. I may need to look into if it's possible to exclude these custom fields from Formie's field builder Vue app parsing.

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/formie:"dev-craft-4 as 2.0.32"`.

**Comment by engram-design** [comment]
Fixed in [2.0.33](https://github.com/verbb/formie/releases/tag/2.0.33)
